### PR TITLE
CCIBI-2597 adding prop for adjusting vertical spacing

### PR DESCRIPTION
**Pull request by MrMuzik**

### CODE REVIEW

**Comment by johnny-five** [comment]
I, @cachapman, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket](https://technekes.atlassian.net/browse/ccibi-2597)
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

File: src/DatePickerRange/DatePickerRange.component.tsx [hunk 1]
```diff
@@ -157,6 +157,7 @@ export class DatePickerRange extends React.PureComponent<Props> {
     invalid: false,
     invalidText: '',
     invalidTextColor: '',
+    verticalSpacing: 10,
```
Comment (cachapman) [inline src/DatePickerRange/DatePickerRange.component.tsx:160]: Have you thought about turning it up to 11? j/k

File: package.json [hunk 2]
```diff
@@ -121,8 +121,8 @@
     "typescript": "3.3.x",
     "uglify-es": "^3.3.9",
     "webpack": "^4.25.1",
-    "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.14",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.11.0",
```
Comment (cachapman) [inline package.json:125]: These seem like big upgrades which may be outside the scope of the story - do they require regression on the app? are there any release notes between those versions?

Comment (MrMuzik) [reply package.json]: I didn't intend for those updates to be made. They are removed now.

**Pull request merged**